fhir-install/Dockerfile

```diff
@@ -52,11 +48,21 @@ ENV FHIR_CONFIG_HOME=/opt/ol/wlp/usr/servers/defaultServer \
     TRACE_FILE=stdout \
     TRACE_FORMAT=BASIC
 
+COPY target/LICENSE /licenses/
```

Comment:
isn't there a root /LICENSE ?

Reply:
this wasn't really a change from before...I just moved it lower for some reason (I think I was grouping the COPYs).  that said, I don't really know that its needed.

Reply:
It does look like the one under target here is a copy of our root LICENSE file:
```
            <plugin>
                <artifactId>maven-antrun-plugin</artifactId>
                <version>3.0.0</version>
                <executions>
                    <execution>
                        <phase>package</phase>
                        <configuration>
                            <target>
                                <copy
                                    todir="${project.build.directory}">
                                    <fileset
                                        dir="${project.build.directory}/../../">
                                        <include name="LICENSE" />
                                    </fileset>
                                </copy>
                            </target>
                        </configuration>
                        <goals>
                            <goal>run</goal>
                        </goals>
                    </execution>
                </executions>
            </plugin>
            ```
            
I think we just need it here so that it exists under the context of the directory being passed to docker (because docker doesn't let you pass file from outside of this context)